bin/init

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# This script will setup the database and then start the rails application
+
+set -e
+# check that BUNDLER_BIN is set
+if [ ! -z $BUNDLER_BIN ]; then
```

Comment:
But this restricts the usage to our `sle` based docker images. I would preferably use `bundler` directly since we know it exists, or that isn't the case on our `sle` based docker images? The idea is to be able to use the same init script for both the `sle` docker images and `opensuse` images.